glide.yaml

```diff
@@ -33,3 +33,7 @@ import:
   version: ^0.3.0
 - package: github.com/vishvananda/netlink
 - package: github.com/gavv/monotime
+- package: github.com/onsi/ginkgo
```

Comment:
As we spoke, I've moved gomega into `testImport` but left ginkgo out due to conflicts.